lntest/harness_miner.go

```diff
@@ -487,3 +495,89 @@ func (h *HarnessMiner) MineEmptyBlocks(num int) []*wire.MsgBlock {
 
 	return blocks
 }
+
+// SpawnTempMiner creates a temp miner and syncs it with the current miner.
+// Once miners are synced, the temp miner is disconnected from the original
+// miner and returned.
+func (h *HarnessMiner) SpawnTempMiner() *HarnessMiner {
+	require := require.New(h.T)
+
+	// Setup a temp miner.
+	tempLogDir := ".tempminerlogs"
+	logFilename := "output-open_channel_reorg-temp_miner.log"
```

Comment:
I guess this could be a bit more generic now as well? Since we're calling this not just from the channel open reorg test?

Reply:
Good catch, changed it to be more generic.